src/main/scala/lisa/proven/mathematics/SetTheory.scala

```diff
@@ -7,7 +7,7 @@ import lisa.automation.kernel.ProofTactics.*
  * An embryo of mathematical development, containing a few example theorems and the definition of the ordered unorderedPair.
  */
 object SetTheory extends lisa.Main {
-
+  /*
```

Comment:
It's great to have the error reporting system; why were many proofs commented out though?